config/locales/account_reset/en.yml

```diff
@@ -33,8 +33,8 @@ en:
       confirm: If you cancel now, you must create a new request and wait another
         %{interval} to delete your account.
       header: You requested to delete your account
-      wait_html: There is a %{hours}-hour waiting period to delete your account. In
-        <strong>%{interval}</strong>, you will receive an email with
+      wait_html: There is a %{waiting_period} waiting period to delete your account.
```

Comment:
I think this might have the same issue @jmdembe identified in her earlier pull request, where the grammar would be wrong in how the plural hours is inserted here, e.g.

"There is a 24 hours waiting period to delete your account."

Instead of "24-hour" or "24 hour"

Since it's causing us trouble, maybe we could also check with UX if it's possible to rephrase this to better support plural unit, e.g. "There is a wait period of 24 hours before you can delete your account"

Reply:
yea. I think that works fine, the issue is we would then need to go through our translation process right?

Reply:
Yeah we would.